muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/evaluation/body/application/evaluation/evaluation.tsx

```diff
@@ -24,12 +24,15 @@ import {
 } from "~/actions/main-function/evaluation/evaluationActions";
 import "~/sass/elements/assignment.scss";
 import "~/sass/elements/empty.scss";
+import { WorkspaceType } from "../../../../../reducers/workspaces/index";
```

Comment:
Use tilde instead of relative path as tilde automatically starts from app root.

Reply:
Fixed in previous commit